Style cleanup

jfrankl opened the pull request:
Style cleanup across application.

![Screen Shot 2021-08-10 at 1 09 31 PM](https://user-images.githubusercontent.com/1809908/128904495-7e65b439-47ad-4967-89fe-0205d7cc467e.png)
![Screen Shot 2021-08-10 at 1 12 41 PM](https://user-images.githubusercontent.com/1809908/128904497-4c6e773a-7eed-4fe7-9045-248d2b10d827.png)
![Screen Shot 2021-08-10 at 1 11 58 PM](https://user-images.githubusercontent.com/1809908/128904498-4683b694-d6bf-4e99-8107-e7042b68088d.png)
![Screen Shot 2021-08-10 at 1 11 37 PM](https://user-images.githubusercontent.com/1809908/128904500-d3c2a51b-182c-4132-932d-269203b71520.png)
![Screen Shot 2021-08-10 at 1 11 24 PM](https://user-images.githubusercontent.com/1809908/128904503-629e92c0-c597-4978-8ffc-dac3a37913dc.png)
![Screen Shot 2021-08-10 at 1 10 03 PM](https://user-images.githubusercontent.com/1809908/128904504-0aa82c93-400c-4141-b199-e2f8c45ca2b1.png)

## Testing Instructions
- This PR makes a number of style fixes across the application, be sure to take a look around in every section, specifically
- The home screen (map list)
- Project screen with the sidebar expanded or not expanded
- Evaluate mode, Competitiveness view
- The find button

Fixes #865

kevinearldenny reviewed (commented):
This mostly looks great! That being said, I do have a few comments (by page below)

### Home screen

- Could we make the maps a little bit bigger? And perhaps apply a bit more padding between them and the project title? The screen feels a bit out of balance to the left currently

### Project screen

#### Sidebar
![image](https://user-images.githubusercontent.com/66973361/128928730-f5280bd4-ac0f-42ae-8852-39813c57e6a5.png)
- I think the expand button looks a little weird in it's new place. Could we move it back over to the corner when not expanded?

#### Find menu

- It looks a little weird not having "Find" written next to the icon IMO

jfrankl commented:
Thanks for taking a look @kevinearldenny.

I made the maps slightly larger and reduced some of the padding around them. I also centered this content and added a little more spacing. I'm trying to strike a balance between wanting to show off the maps, but also make it easy for users to see several projects and quickly scroll down the page. The larger the maps, the longer the page. I think this is looking better.

![Screen Shot 2021-08-11 at 7 31 58 AM](https://user-images.githubusercontent.com/1809908/129021705-ff8063c5-224b-4cca-b636-bd927b822820.png)

I hear you about the "Find" text for the search. The reason I'm removing that label is to preserve space in the Map Header. I want to make sure the application works well on a variety of screen sizes, and I think this is a good tradeoff. I expect the search will still be discoverable enough given its relative importance, but I'm open to reconsidering this in the future if we learn otherwise. In the meantime, I added a tooltip and slightly cleaned up the hover state.

![Screen Shot 2021-08-11 at 7 32 52 AM](https://user-images.githubusercontent.com/1809908/129021812-3371fa11-6edf-4446-8a0c-ec5a7667b866.png)

This choice for the Expand Sidebar button location is so when the sidebar is expanded but is too wide to fit on a user's screen, they can still see the button so they can collapse the sidebar. Before I made this change, a user would need to scroll to the right before they could see the button.
![Screen Shot 2021-08-11 at 7 29 47 AM](https://user-images.githubusercontent.com/1809908/129021472-11a7e96c-c3d5-4e46-bc1c-90aeb6a27778.png)
